lib/Conversion/TorchToTMTensor/TorchToTMTensor.cpp

```diff
@@ -541,19 +541,9 @@ class ConvertAtenBincountOp : public OpConversionPattern<AtenBincountOp> {
 
 namespace {
 
-Value combinePutIndices(Location loc, llvm::ArrayRef<Value> indicesRef,
-                        OpBuilder b) {
-  llvm::SmallVector<Value> indices(indicesRef);
-  // Declare commonly used constants up front:
-  Value torchCstZero =
-      b.create<Torch::ConstantIntOp>(loc, b.getI64IntegerAttr(0));
-  Value torchCstOne =
-      b.create<Torch::ConstantIntOp>(loc, b.getI64IntegerAttr(1));
-  Value torchCstNegOne =
-      b.create<Torch::ConstantIntOp>(loc, b.getI64IntegerAttr(-1));
-
-  // Determine the broadcast sizes and materialize missing implicit end
-  // dimensions:
+// Determine the common shape to broadcast all the index tensors to.
```

Comment:
How about:
"Determine the common broadcast shape of all the index tensors"

just to avoid the to at the end?